vopono_core/src/config/providers/protonvpn/openvpn.rs

```diff
@@ -182,15 +182,13 @@ impl OpenVpnProvider for ProtonVPN {
 #[derive(EnumIter, PartialEq)]
 enum Tier {
     Plus,
-    Basic,
     Free,
 }
 
 impl Tier {
     fn url_part(&self) -> String {
         match self {
             Self::Plus => "2".to_string(),
-            Self::Basic => "1".to_string(),
             Self::Free => "0".to_string(),
```

Comment:
Not sure whether I have to adjust the numbers here, as I'm not a usual project maintainer.
Building and running it worked.